cloud-deployment.md

```diff
@@ -6,8 +6,9 @@ toc: false
 
 Use the following guides to deploy CockroachDB to popular cloud platforms:
 
-- [Google Cloud Platform (GCE)](deploy-cockroachdb-on-google-cloud-platform.html)
 - [Amazon Web Services (AWS)](deploy-cockroachdb-on-aws.html)
+- [Digital Ocean Deployment](deploy-cockroachdb-on-digital-ocean.html)
```

Comment:
Nit: remove "Deployment", since we're just listing the names of various cloud platforms.